Documentation/writing-tizen-native-apps.md

```diff
@@ -0,0 +1,86 @@
+# Writing a Tizen Native App
+
+This document guides how to write a Tizen Native (C/C++) app with nnstreamer APIs (Tizen Machine-Learning Inference API Set).
+
+## About the Tizen ML-Inference API Sets
+
+There are two API sets as of Tizen 5.5 M2, Machine-Learning Inference: "Pipeline" and "Single".
+
+With **Pipeline** API set, you can construct a GStreamer pipeline with nnstreamer elements and gstreamer elements in the "whitelist". GStreamer plugins not listed in the "whitelist" is not supported by Pipelie APIs. Note that an alias element, ```tizencamvideosrc``` (or ML_TIZEN_CAM_VIDEO_SRC macro), denotes for a video camera source, which may vary per device; in other words, you don't need to worry whether it's ```v4l2src``` or ```camsrc``` or whatsoever. For the list of whitelist, refer to the Tizen API doc or ```/etc/nnstreamer.ini```.
```

Comment:
Changes marked as bold in the paragraph

With Pipeline API set, you can construct a GStreamer pipeline with nnstreamer elements and gstreamer elements in the "whitelist". GStreamer plugins not listed in the "whitelist" **~is~ -> are** not supported by Pipelie APIs. Note that an alias element, ```tizencamvideosrc``` (or ML_TIZEN_CAM_VIDEO_SRC macro), denotes **~for~** a video camera source, which may vary per device; in other words, you don't need to worry whether it's ```v4l2src``` or ```camsrc``` or whatsoever. For the list of whitelist **elements**, refer to the Tizen API doc or ```/etc/nnstreamer.ini```.

Reply:
Fixed.